src/host/screenInfo.cpp

```diff
@@ -103,10 +103,10 @@ SCREEN_INFORMATION::~SCREEN_INFORMATION()
     try
     {
         IWindowMetrics* pMetrics = ServiceLocator::LocateWindowMetrics();
-        THROW_IF_NULL_ALLOC(pMetrics);
+        THROW_HR_IF_NULL(E_FAIL, pMetrics);

```

Comment:
I don't think this should throw `E_INVALIDARG`. So I choose `E_FAIL`. Any advice?

Reply:
That seems right to me